### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, lstat 'C:\Users\peter\AppData\Local\atom\app-1...

### ISSUE THREAD

**Issue by petereast** [atom/fuzzy-finder#224]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.8.0
**System**: Microsoft Windows 10 Home Single Language
**Thrown From**: [fuzzy-finder](https://github.com/atom/fuzzy-finder) package, v1.0.5

### Stack Trace

Uncaught Error: ENOENT: no such file or directory, lstat 'C:\Users\peter\AppData\Local\atom\app-1.8.0\atom:'

```
At ATOM_SHELL_ASAR.js:0

Error: ENOENT: no such file or directory, lstat 'C:\Users\peter\AppData\Local\atom\app-1.8.0\atom:'
    at Error (native)
    at fs.lstatSync (fs.js:887:18)
    at Object.fs.lstatSync (ATOM_SHELL_ASAR.js:229:16)
    at Object.realpathSync (fs.js:1532:21)
    at Object.fs.realpathSync (ATOM_SHELL_ASAR.js:304:29)
    at C:\Users\peter\AppData\Local\atom\app-1.8.0\resources\app.asar\node_modules\fuzzy-finder\lib\path-loader.js:19:21
    at Array.map (native)
    at Object.module.exports.startTask (C:\Users\peter\AppData\Local\atom\app-1.8.0\resources\app.asar\node_modules\fuzzy-finder\lib\path-loader.js:17:46)
    at ProjectView.module.exports.ProjectView.runLoadPathsTask (C:\Users\peter\AppData\Local\atom\app-1.8.0\resources\app.asar\node_modules\fuzzy-finder\lib\project-view.js:190:46)
    at ProjectView.module.exports.ProjectView.populate (C:\Users\peter\AppData\Local\atom\app-1.8.0\resources\app.asar\node_modules\fuzzy-finder\lib\project-view.js:111:21)
    at ProjectView.module.exports.ProjectView.toggle (C:\Users\peter\AppData\Local\atom\app-1.8.0\resources\app.asar\node_modules\fuzzy-finder\lib\project-view.js:87:14)
    at atom-workspace.atom.commands.add.fuzzy-finder:toggle-file-finder (C:\Users\peter\AppData\Local\atom\app-1.8.0\resources\app.asar\node_modules\fuzzy-finder\lib\main.js:9:46)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\peter\AppData\Local\atom\app-1.8.0\resources\app.asar\src\command-registry.js:260:29)
    at C:\Users\peter\AppData\Local\atom\app-1.8.0\resources\app.asar\src\command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (C:\Users\peter\AppData\Local\atom\app-1.8.0\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (C:\Users\peter\AppData\Local\atom\app-1.8.0\resources\app.asar\node_modules\atom-keymap\lib\keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (C:\Users\peter\AppData\Local\atom\app-1.8.0\resources\app.asar\src\window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (C:\Users\peter\AppData\Local\atom\app-1.8.0\resources\app.asar\src\window-event-handler.js:3:61)
```

### Commands

```
  3x -1:38.8.0 core:backspace (atom-text-editor.editor.is-focused)
     -1:31.1.0 editor:newline (atom-text-editor.editor.is-focused)
 10x -1:28.2.0 core:backspace (atom-text-editor.editor.is-focused)
     -1:25.7.0 core:save (atom-text-editor.editor.is-focused)
     -1:14.6.0 editor:newline (atom-text-editor.editor.is-focused)
 19x -1:12.8.0 core:backspace (atom-text-editor.editor.is-focused.autocomplete-active)
     -0:29.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:06.1.0 grammar-selector:show (atom-text-editor.editor)
     -0:04.1.0 fuzzy-finder:toggle-file-finder (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "one-light-ui",
      "one-light-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
minimap-hide, v0.3.0

# Dev
No dev packages
```

**Comment by KuSh** [comment]
Hi,

Same error here, steps to reproduce:

1. Hit CTRL-SHIFT T (I thought I was on firefox), while reading the readme of an uninstalled package (atom-maven) in the settings panel

**Atom Version**: 1.8.0
**System**: linux 4.6.0-1-amd64
**Thrown From**: [fuzzy-finder](https://github.com/atom/fuzzy-finder) package, v1.0.5


### Stack Trace

Uncaught Error: ENOENT: no such file or directory, lstat '/home/nlecam/devel/tools/atom:'

```
At ATOM_SHELL_ASAR.js:0

Error: ENOENT: no such file or directory, lstat '/home/nlecam/devel/tools/atom:'
    at Error (native)
    at fs.lstatSync (fs.js:887:18)
    at Object.fs.lstatSync (ATOM_SHELL_ASAR.js:229:16)
    at Object.realpathSync (fs.js:1532:21)
    at Object.fs.realpathSync (ATOM_SHELL_ASAR.js:304:29)
    at /home/nlecam/devel/tools/atom/resources/app.asar/node_modules/fuzzy-finder/lib/path-loader.js:19:21
    at Array.map (native)
    at Object.module.exports.startTask (/home/nlecam/devel/tools/atom/resources/app.asar/node_modules/fuzzy-finder/lib/path-loader.js:17:46)
    at ProjectView.module.exports.ProjectView.runLoadPathsTask (/home/nlecam/devel/tools/atom/resources/app.asar/node_modules/fuzzy-finder/lib/project-view.js:190:46)
    at ProjectView.module.exports.ProjectView.populate (/home/nlecam/devel/tools/atom/resources/app.asar/node_modules/fuzzy-finder/lib/project-view.js:111:21)
    at ProjectView.module.exports.ProjectView.toggle (/home/nlecam/devel/tools/atom/resources/app.asar/node_modules/fuzzy-finder/lib/project-view.js:87:14)
    at atom-workspace.atom.commands.add.fuzzy-finder:toggle-file-finder (/home/nlecam/devel/tools/atom/resources/app.asar/node_modules/fuzzy-finder/lib/main.js:9:46)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/home/nlecam/devel/tools/atom/resources/app.asar/src/command-registry.js:260:29)
    at /home/nlecam/devel/tools/atom/resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/home/nlecam/devel/tools/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/home/nlecam/devel/tools/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/home/nlecam/devel/tools/atom/resources/app.asar/src/window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (/home/nlecam/devel/tools/atom/resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -1:11.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:00.6.0 fuzzy-finder:toggle-file-finder (a.)
     -0:00.3.0 core:focus-next (a.)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "metrics"
    ]
  }
}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

**Comment by 50Wliu** [comment]
This should be fixed in Atom 1.9.0-beta0 and above.
